File: Python/03. Code Related to Arrays/61. Find the frequency of element in an array.py
lengthOfArray = int(input("Enter the length of an Array:"))
array = []

for i in range(lengthOfArray):
    element = int(input(f"Enter {i+1} element :"))
    array.append(element)

size = len(array)
for i in range(size):
        flag = False
        count = 0

        for j in range(i+1, size):
            if array[i] == array[j]:
                flag = True
                break

        if flag == True:
            continue

        for j in range(0, i+1):
            if array[i] == array[j]:
                count += 1

        print("{0}: {1}".format(array[i], count))

# Method 2

# freq={}
# for i in array:
#     if i in freq:
#         freq[i] += 1
#     else:
#         freq[i] = 1

#     # Method 3
#     # freq[i]=freq.get(i,0)+1
# print(freq)

File: Java/03. Code Related to Arrays/61. Find the frequency of element in an array.java
import java.util.*;

class Main {
    public static void main(String [] args) {
        Scanner userInput = new Scanner(System.in);

        System.out.print("Enter the number of elements of the Array: ");
        int size = userInput.nextInt();
        int[] numbers = new int[size];

        for (int i = 0; i < size; i++) {
            System.out.print("Element " + (i + 1) + ": ");
            numbers[i] = userInput.nextInt();
        }

        int n = numbers.length;
        for (int i = 0; i < n; i++) {
            Boolean flag = false;
            int count = 0;

            for (int j = i+1; j < n; j++) {
                if (numbers[i] == numbers[j]) {
                    flag = true;
                    break;
                }
            }
            if (flag == true) {
                continue;
            }

            for (int j = 0;  j <= i; j++){
                if (numbers[i] == numbers[j]) {
                    count += 1;
                }
            }
            System.out.println(numbers[i]+":" +count);
        } 
        userInput.close();
    }
}